### PR TITLE
Fix Stylus variables match

### PR DESCRIPTION
**Pull request by tomaszbakula**
Fix the issue #197

### CODE REVIEW

File: lib/modules/SmartVariable.coffee [hunk 1]
```diff
@@ -8,7 +8,7 @@
     # -------------------------------------
     #  Variable Types
     # -------------------------------------
-        VARIABLE_PATTERN = '\\{{ VARIABLE }}[\\s]*\\:[\\s]*([^\\;\\n]+)[\\;|\\n]'
+        VARIABLE_PATTERN = '\\{{ VARIABLE }}[\\s]*[:|=][\\s]*([^\\;\\n]+)[\\;|\\n]'
```
Comment (thomaslindstrom) [inline lib/modules/SmartVariable.coffee]: There `|` has no use inside a bracket (it doesn't mean `OR`, as inside a capturing group): In this case, you'd also, unintentionally, match
```css
$variable|value
```

Comment (thomaslindstrom) [inline lib/modules/SmartVariable.coffee]: Thanks for the PR!

Comment (tomaszbakula) [inline lib/modules/SmartVariable.coffee]: You're right. There shouldn't be `|`. I've updated this.

**Comment by thomaslindstrom** [comment]
Thanks, @tomaszbakula!

**Pull request merged**